pages/dkp/kommander/1.3/release-notes/1.3.3/index.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: Release Notes Kommander 1.3.3
+title: Release Notes Kommander 1.3.3
+menuWeight: 40
+excerpt: View release-specific information for Kommander 1.3.3
+enterprise: false
+---
+
+**D2iQ&reg; Kommander&reg; version 1.3.3 was released on x, May 2021.**
+
+[button color="purple" href="https://support.d2iq.com/s/entitlement-based-product-downloads"]Download Konvoy[/button]
+
+To get started with Kommander, [download](/dkp/konvoy/latest/download/) and [install](/dkp/konvoy/latest/install/) the latest version of Konvoy.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+# Release Summary
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
+
+# Known issues
+
+In versions prior to 1.3.3, Kubecost cost-analyzer by default requests a PVC of size 0.2Gi. This may cause issues with certain provisioners that require a minimum storage size of 1Gi. To resolve this issue, upgrade to 1.3.3 following the upgrade instructions below.
+
+## Upgrading
+The default cost-analyzer persistent volume size has been increased to 32Gi. Upgrades may require manual intervention if your provisioner does not support volume expansion. Here are some recommendations from the [Kubecost storage documentation](https://github.com/kubecost/docs/blob/master/storage.md).
+
+**If you are upgrading an existing version of Kommander**
+  - If your provisioner does supports volume expansion, we will automatically resize you to a 32GB disk in an upgrade.
+  - If your provisioner does not support volume expansion:
+    - If you have not added or modified Kubecost configuration from the settings page on the Kubecost frontend, you can safely delete the PV and upgrade.
+      - We suggest you delete the old PV, then upgrade Kommander.
+    - If you cannot safely delete the PV storing your configs and configure them on a new PV:
+      - If you are not on a regional cluster, we recommend that you provision a second PV by setting `kubecost.cost-analyzer.persistentVolume.dbPVEnabled=true` in your cluster.yaml under Kommander values.
```

Comment:
I'm having a lot of troubles following the bullets here, all the negative conditions make it very difficult to understand.

It seems like there are two major cases here:

1. You have not modified the default configs, in which case you should just delete the PV and then upgrade.  (Where do we explain how to find the right volume?)

If this is correct, this sentence seems like it would cover it:
   
    - If you have not added or modified the Kubecost configuration using the settings page on the Kubecost frontend, you can safely delete the PV and upgrade.  We suggest you delete the old PV first, and then upgrade Kommander.   (But - we need to add to this HOW to find the right PV.  Also - what happens if you don't delete the old PV before upgrade?)

2. You've modified the configs.   This is where it gets hard to follow what to do.   Is it possible to backup/restore them to a new PV?  If so, how? If not, what happens to my settings, do I have to redo them? 

I think we should write out all the scenarios here in long form and *then* try to condense it down to a bullet list.

Finally, it seems that Kubecost now has support for topology aware storage, but we only recommend using that if you have already customized the Kubecost settings and only for a 'second' PV?   The mention of a second PV is confusing to me.  

Also, this seems like a separate, new feature, which is independent of this one particular upgrade scenario.   Is this something I can use with a new installation?  If so, how?   Finally, let's make sure we use the same terminology as the K8S reference page here.  'Regional Cluster' isn't a term used anywhere else in our docs.  It seems like we mean "multi-zone cluster"?